tomee/tomee-microprofile/mp-common/src/main/java/org/apache/tomee/microprofile/TomEEMicroProfileListener.java

```diff
@@ -66,6 +64,8 @@ public class TomEEMicroProfileListener {
         "io.smallrye.faulttolerance.FaultToleranceExtension",
         };
 
+    private static final Map<AppInfo, Index> indexCache = new HashMap<>();
```

Comment:
guess it can not be static since it belongs to the listener instance

Reply:
removed static, it's indeed not needed